jrnl/jrnl.py

```diff
@@ -168,8 +171,27 @@ def search_mode(args, journal, **kwargs):
 
     # Where do the search results go?
     if args.edit:
+        # If we want to both edit and change time in one action
+        if args.change_time:
+            # Generate a new list instead of assigning so it won't be
+            # modified by _change_time_search_results
+            selected_entries = [e for e in journal.entries]
+
+            no_change_time_prompt = len(journal.entries) == 1
```

Comment:
This doesn't work as intended right now. I think this line is supposed to reference `selected_entries` instead of `journal.entries`.

Reply:
@wren, can you clarify the bug you're seeing? `selected_entries` here is built from `journal.entries`, so they have the same length. `selected_entries` only exists so we can reset `journal.entries` to the original set of entries on line 188.

(I agree that a test should cover the functionality--just added one.)

Reply:
Hi, @richardjs. Back from vacation now; thanks for your patience!

Nevermind about this bug! I misinterpreted the requirement about when to prompt the user. I thought we were always skipping when there was only one entry selected, but I see now that it's only when the edit flag is also used. So, this works as expected. Thank you!